build.gradle

```diff
@@ -981,8 +982,14 @@ project('standalone') {
         if (systemProperties.get("extDirs") != null) {
             classpath += files(systemProperties.get("extDirs"))
         }
-
-        jvmArgs "-Xdebug", "-Xrunjdwp:server=y,transport=dt_socket,address=8000,suspend=n"
+        
+        def jvmDebugArgs=System.getenv("JVM_DEBUG_ARGS")
+        if(jvmDebugArgs !=null&& jvmDebugArgs !=""){
+            jvmArgs "-Xdebug", jvmDebugArgs
```

Comment:
I think we probably want a default if the user doesn't specify. (Maybe not 8000)

Reply:
Sure, Retaining default behavior of Pravega-Standalone where default debug port(hardcoded) is enabled and allowing user to provide the custom one through environment variable.